### PR TITLE
[SASS-3696] Upgraded integration test for page: 'Pensions where tax relief is not claimed'.

### PR DESCRIPTION
**Pull request by erasmos-hmrc**
# Checklist

Jira subtask: SASS-3697

## Developer: Before Asking Others To Review Your Code

Have you ...

- [X]  Had a QA verify that all acceptance criteria have been met, with comment(s) in the ticket? 
- [X]  Hosted a group review of your implementation?
- [X]  Rebased against `main`? 
- [X]  Ensured that your commit(s) contain the ticket key?
- [X]  Ensured that your branch is named after the ticket key?
- [X]  Successfully run the unit tests?
- [X]  Successfully run the integration tests?
- [ ]  Successfully run the journey tests? If not, please add a comment explaining why it wasn't necessary.
- [x]  Ensured that the PR job has passed.

## Reviewer: Before Reviewing

Have you ...

- [ ]  Pulled the branch down?
- [ ]  Assigned yourself to this PR?
- [ ]  Moved the Jira *subtask* to “in review”?
- [ ]  Ensured that it is rebased against `main`?

## Developer: After Approval

Have you ...

- [ ]  Rebased against `main`?
- [ ]  Moved the Jira *subtask* to “in pipeline”?

## Developer: After Merging

Have you ...

- [ ]  Deleted the branch?
- [ ]  Scheduled a demo? If not, please add a comment.

### CODE REVIEW

**Comment by erasmos-hmrc** [comment]
> Successfully run the journey tests? If not, please add a comment explaining why it wasn't necessary.

I only changed the tests.

File: it/controllers/pensions/paymentsIntoPensions/PensionsTaxReliefNotClaimedControllerISpec.scala [hunk 1]
```diff
@@ -16,257 +16,780 @@
...
+        "the user has a session with an empty pensions model" in {
+
+          val response = getPage(Some(pensionsUserData(aPensionsCYAEmptyModel)))
+
+          response must haveStatus(SEE_OTHER)
+          response must haveALocationHeaderValue(PageRelativeURLs.paymentsIntoPensionsReliefAtSourcePage)
+
+        }
+        "the user has a session with an insufficient pensions model" when {
+          "case one" in {
```
Comment (abiodxn) [inline it/controllers/pensions/paymentsIntoPensions/PensionsTaxReliefNotClaimedControllerISpec.scala:66]: I thnk we should expand on these case names for clarity

Comment (erasmos-hmrc) [reply it/controllers/pensions/paymentsIntoPensions/PensionsTaxReliefNotClaimedControllerISpec.scala:66]: OK. I'll make that change now.

Comment (erasmos-hmrc) [reply it/controllers/pensions/paymentsIntoPensions/PensionsTaxReliefNotClaimedControllerISpec.scala]: Changed.

**Review by abiodxn — APPROVED** [review]

**Pull request merged**